django-stubs/core/mail/message.pyi

```diff
@@ -78,26 +83,26 @@ class EmailMessage:
     def attach(
```

Comment:
If you're feeling adventurous, you can have a more precise type here by using an `@overload` -- first case is `filename: MIMEText` and second case is the others.

Same for `_AttachmentTuple` (union of the two options).